js/modules/k6/ws/ws_test.go

```diff
@@ -463,7 +467,13 @@ func TestTLSConfig(t *testing.T) {
 		Group:  root,
 		Dialer: tb.Dialer,
 		Options: lib.Options{
-			SystemTags: lib.GetTagSet("url", "proto", "status", "subproto", "ip"),
+			SystemTags: stats.ToSystemTagSet([]string{
```

Comment:
Again, this seems unnecessary, a simple `|` of the enabled tags should be enough. Or alternatively, a helper function that doesn't require strings

Reply:
@na-- We do need because we get user input, or env var as list of string:

```
if flags.Changed("system-tags") {
		systemTagList, err := flags.GetStringSlice("system-tags")
		if err != nil {
			return opts, err
		}
		opts.SystemTags = stats.ToSystemTagSet(systemTagList)
	}
```

Reply:
Current code https://github.com/loadimpact/k6/blob/ce85f802fe139b36e41560c5acdd0478ab3eb375/cmd/options.go#L131

Reply:
Sure, but as I explain in https://github.com/loadimpact/k6/pull/1148#discussion_r334862919, long term (i.e. not in this PR), any user input would need to be validated, so in the future the function signature of `ToSystemTagSet()` would probably have to be something like this:
```go
func ToSystemTagSet(tags []string) (*SystemTagSet, error)
```

And as I mentioned in https://github.com/loadimpact/k6/pull/1148#discussion_r334874715, we can add a super simple constructor/helper function that just accepts a variable number of tags and returns a single `*SystemTagSet`. That would be easy to use, type safe, and with no possibility of error since since we can only pass other system tags as parameters

Reply:
@na-- Fair enough, added.